src/flake_info.rs

```diff
@@ -236,6 +241,10 @@ impl FlakeMetadata {
     }
 
     pub async fn outputs(&self, include_output_paths: bool) -> Result<FlakeOutputs> {
+        if self.single_system_evaluation {
+            return Ok(FlakeOutputs(serde_json::json!({})));
+        }
```

Comment:
This should probably have a different name than "single system evaluation" -- maybe `--large-evaluation` or something?

Reply:
Or probably better: `--skip-outputs` or similar

Reply:
originally itwas called `gigantic` ?